db/migrate/20230602181048_create_constraints.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+class CreateConstraints < ActiveRecord::Migration[7.0]
+  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/AbcSize
+  def change
+    create_table :constraints do |t|
+      t.string :name, null: false
+      t.string :category, null: false
+
+      t.index [:name], unique: true
```

Comment:
Could remove this if we are adding a unique index for the combination of local authority and name instead

Reply:
yes good shout!